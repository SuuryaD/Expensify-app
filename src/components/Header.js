import React from 'react';
import {BrowserRouter , Route, Switch ,Link, NavLink} from 'react-router-dom';

const Header = () => (
    <div>
        <h1>Expensify App</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Home </NavLink>
        <NavLink to="/create" activeClassName="is-active">create Expense </NavLink>
    </div>
);

export default Header;
