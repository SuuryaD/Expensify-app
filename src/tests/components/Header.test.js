import { shallow } from 'enzyme';
import Header from '../../components/Header';
import React from 'react';


test('Should render header correctly', () => {
    const wrapper = new shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});
