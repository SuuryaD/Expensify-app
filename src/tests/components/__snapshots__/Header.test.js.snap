// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header correctly 1`] = `
<div>
  <h1>
    Expensify App
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home 
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    create Expense 
  </[object Object]>
</div>
`;
